adbmanager/adbmanager_test.go

```diff
@@ -0,0 +1,77 @@
+package adbmanager
+
+import (
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/stretchr/testify/require"
+	"testing"
+
+	"github.com/bitrise-io/go-utils/v2/command"
+)
+
+func TestModel_InstallAPKCmd(t *testing.T) {
```

Comment:
I suggest using Given When Then test function naming.
I think we agreed upon this but missed adding it to our documentation. Here are some examples: https://github.com/bitrise-steplib/steps-xcode-test/blob/master/output/output_test.go#L27

Reply:
I propose: `Test_GivenAPKPath_WhenCreateInstallAPKCmd_ThenCreatesExpectedCommand`